examples/Query_example-Finnish_PIC.xml

```diff
@@ -116,9 +116,9 @@
             </urn1:AppHdr>
             <urn2:Document>
                 <urn2:InfReqOpng>
-                    <urn2:InvstgtnId>ABC1234</urn2:InvstgtnId>
+                    <urn2:InvstgtnId>Customs_aggr</urn2:InvstgtnId>
```

Comment:
Should we also update the investigation start date to 2020-09-01 while we are updating these? Does it matter that the signature does not match now after these updates, or should we update that also?

Reply:
Updated dates, left signatures - they are just examples. Can update later if really needed